vmm/src/config.rs

```diff
@@ -1930,9 +1930,18 @@ impl VmConfig {
             }
         }
 
+        // The 'conflict' check is introduced in commit 24438e0390d3
+        // (vm-virtio: Enable the vmm support for virtio-console).
+        //
+        // Allow double tty mode, for someone want to use virtio console
+        // for better performance, and want to keep legacy serial to catch
+        // boot stage logs for debug.
+        // Using such double tty mode, you need to configure the kernel
+        // properly, such as:
+        // "console=hvc0 earlyprintk=ttyS0"
         if self.console.mode == ConsoleOutputMode::Tty && self.serial.mode == ConsoleOutputMode::Tty
         {
-            return Err(ValidationError::DoubleTtyMode);
+            info!("{}", ValidationError::DoubleTtyMode);
```

Comment:
I think it is better to say something like: `Both serial and console are being set as TTY mode. Guest kernel command-line should include "console=hvc0 earlyprintk=ttyS0".`

Reply:
I don't think we should add linux kernel command-line suggestion in CH logs（maybe in doc or comments）, so just change the error return into some logs is enough.